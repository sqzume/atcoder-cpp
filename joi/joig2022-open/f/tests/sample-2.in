10 9 3 25
1 2 2
2 3 1
3 4 1
4 5 1
5 6 2
6 7 1
7 8 1
8 9 1
9 10 2
10
9
3
