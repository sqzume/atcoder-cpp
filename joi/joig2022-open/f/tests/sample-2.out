Large
22
4
